### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by maxonary**

### CODE REVIEW

**Comment by koenidv** [comment]
Much text, many wow :) You're referencing a `CITATION.cff` file that you did not include in the PR. Also, would be cool to keep the Netlify build badge on top.

**Comment by koenidv** [comment]
Any updates on this?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,56 @@
-# PawnHub
+# Landing page - Remote Chess Experience
```
Comment (koenidv) [inline README.md]: Can you make clear that this project contains not only the landing page but also the web client for the project? Thanks 💙

**Comment by maxonary** [comment]
The requirements have been fulfilled. 
I request the merge to main.

**Review by koenidv — APPROVED** [review]

**Pull request merged**